libs/authorizenet/src/authorize-net.module.ts

```diff
@@ -0,0 +1,27 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+import { DaffAuthorizeNetStateModule } from './authorize-net-state.module';
+
+const ACCEPT_LIBRARY = 'https://jstest.authorize.net/v1/Accept.js';
```

Comment:
This uses the `sandbox` version of the library, we need a way to configure the production version.

Reply:
I'm going to add a config object, but all the work around that is in the paypal PR, so can we push this PR without this change, then push the paypal, then I'll come back to this?